drools/src/main/java/com/datatorrent/drools/rules/AbstractRulesReader.java

```diff
@@ -0,0 +1,27 @@
+package com.datatorrent.drools.rules;
+
+import java.io.IOException;
+
+import org.apache.hadoop.classification.InterfaceStability.Evolving;
+
+@Evolving
+public interface AbstractRulesReader
```

Comment:
As it is an interface we don't need to have "Abstract" in the name. Every interface is implicitly abstract.